docs/source/cmake_cplusplus.rst

```diff
@@ -0,0 +1,58 @@
+A Simple C++/CMake Example
+==========================
+
+Building a C++ SimpleITK application is normally done using `CMake <https://cmake.org>`_ to
```

Comment:
"is normally done using" -> "is supported by using"

Need to make it a little stronger that this is THE way to do it.